client/my-sites/people/viewers-list/viewers.jsx

```diff
@@ -83,6 +84,10 @@ class Viewers extends Component {
 					: this.props.translate( "You don't have any viewers yet." ),
 		};
 
+		if ( this.props.site && ! this.props.site.jetpack ) {
```

Comment:
Seems like this condition is repeated a few lines above, this looks like a good opportunity to extract to a constant and reuse that.

Reply:
👍 I've created `isJetpackSite` in [0401bc5](https://github.com/Automattic/wp-calypso/pull/63748/commits/0401bc5726a8a5e8cf55ce16930b64cccd8026c5).